admin/inc/functions_view_manager.php

```diff
@@ -64,7 +64,7 @@ function view_manager($base_url, $type, $fields, $sort_options=array(), $conditi
 			{
 				$mybb->input['fields'] = explode(",", $mybb->input['fields_js']);
 			}
-			if(is_array($mybb->input['fields']) && count($mybb->input['fields']) <= 0)
+			if(!is_array($mybb->input['fields']) || (is_array($mybb->input['fields']) && count($mybb->input['fields']) <= 0))
```

Comment:
I think the second `is_array` check is not needed because if `$mybb->input['fields']`:
- is an array the first condition will be false so the second `is_array` will return true again
- is not an array the first condition will be true and second check short-circuited 

```suggestion
			if(!is_array($mybb->input['fields']) || count($mybb->input['fields']) <= 0)
```